### PR TITLE
comment/reply edit and delete functions added

### PR DESCRIPTION
**Pull request by yskmt**
There are some common parts in `add_comment`, `edit_comment` and `delete_comment`, which can be refactored. Also, sometimes the comment deletion and edition are not immediately updated. Is it a PRAW problem?

### CODE REVIEW

**Comment by michael-lazar** [comment]
My guess would be that Reddit is caching the submission on the server side and only updates the cache every so often. 

I agree there is some work that could be done to refactor this. There is a lot of code that will be shared between the SubmissionPage and the SubredditPage.

**Comment by michael-lazar** [comment]
For deleting, how about a confirmation prompt
```
Are you sure you want to delete this post? (y/n):
```

File: rtv/submission.py [hunk 1]
```diff
@@ -112,9 +119,74 @@ def add_comment(self):
...
+        curses.endwin()
+        comment_text = open_editor(comment_info)
+        curses.doupdate()
+
+        if not comment_text:
+            curses.flash()
+            return
+        try:
+            if data['type'] == 'Submission':
+                data['object'].edit(comment_text)
```
Comment (Brobin) [inline rtv/submission.py]: Redundant?

```python
data['object'].edit(comment_text)
```

**Comment by yskmt** [comment]
In order to rebase to the newest master, I will close this pull request.

**Pull request closed without merging**